docs/sources/auth/_index.md

```diff
@@ -26,118 +28,3 @@ Provider | Support | Role mapping | Team sync<br> *(Enterprise only)* | Active s
 [LDAP]({{< relref "ldap.md" >}})                   | v2.1+ | v2.1+ | v5.3+ | v6.3+
 [Okta OAuth]({{< relref "okta.md" >}})             | v7.0+ | v7.0+ | v7.0+ | - 
 [SAML]({{< relref "../enterprise/saml.md" >}}) (Enterprise only)    | v6.3+ | v7.0+ | v7.0+ | - 
-
-## Grafana Auth
```

Comment:
Where did all this content go?

Reply:
New topic added, as mentioned in commit comment.